dashboard/test/controllers/projects_controller_test.rb

```diff
@@ -248,4 +248,17 @@ def sign_in_with_request(user)
     get :load, params: {key: 'gamelab'}
     assert_redirected_to_sign_in
   end
+
+  test 'project validators can go to /featured' do
+    @project_validator = create :teacher
+    @project_validator.permission = UserPermission::PROJECT_VALIDATOR
+    sign_in @project_validator
+    get :featured
+    assert_response :success
+  end
+
+  test '/featured gets redirected to /public if not project validator' do
+    get :featured
```

Comment:
this is a good test for non-signed in users, but it might be good to also add a test for signed in, but not `project_validator`

Reply:
Good idea - will do!

Reply:
Test set up has a signed in user without permissions.  I added one with a signed out user to ensure they're redirected to sign in.